openmetadata-service/src/main/resources/elasticsearch/en/test_case_index_mapping.json

```diff
@@ -406,6 +406,27 @@
           }
         }
       },
+      "tags": {
```

Comment:
we are mapping the tags from the parent table right? I think for the starters we should just use Tier. Make it easy to maintain the propagation here. Tier changes are less frequent

Reply:
Yep, we'll have that as an inherited field

>  Make it easy to maintain the propagation here
what do you mean by this? We added this test [here](https://github.com/open-metadata/OpenMetadata/blob/ae7eb4944cf98dddcc27717af51291f413ccee93/openmetadata-service/src/test/java/org/openmetadata/service/resources/dqtests/TestCaseResourceTest.java#L853) to make sure the updates are propagated correctly.

Reply:
usually column tags can be a quite a bit so was thinking out loud. Another use-case could be propagate glossary terms as well. Users are asking for managing tests via glossary terms, where this may be helpful. If we are propagating tags field it should do the glossary terms as part of it. So this is good to have

Reply:
I'll open a different PR for Glossary terms if that is OK so we don't block the UI to ship that in 1.4.2 if that is ok.